core/editor/loader/v2/editor-v2-loader.php

```diff
@@ -15,19 +18,31 @@ class Editor_V2_Loader extends Editor_Base_Loader {
 	const ENV_PACKAGE = 'env';
 
 	/**
-	 * Packages that should be enqueued (the main app and the extensions of the app).
+	 * Packages (extensions) that should be enqueued based on the active experiments.
 	 */
 	const PACKAGES_TO_ENQUEUE = [
-		// App
-		self::APP_PACKAGE,
-
-		// Extensions
-		'editor-app-bar',
-		'editor-documents',
-		'editor-panels',
-		'editor-responsive',
-		'editor-site-navigation',
-		'editor-v1-adapters',
+		Editor_V2_Experiments::APP_BAR => [
+			'editor-app-bar',
+			'editor-documents',
+			'editor-panels',
+			'editor-responsive',
+			'editor-site-navigation',
+			'editor-v1-adapters',
+		],
+		Editor_V2_Experiments::ELEMENTS => [
```

Comment:
Why do we have to include the packages that are required for both experiments in both lists?
Isn't the app bar a dependency of the elements?

Reply:
no